src/storage/engine/mod.rs

```diff
@@ -231,6 +233,8 @@ mod tests {
                 .unwrap();
             must_put_cf(e.as_ref(), "cf", b"k", b"v1");
         }
+        // Sleep a while to make sure the `LOCK` file is deleted.
+        thread::sleep(Duration::from_millis(100));
```

Comment:
If it happens because of the threadpool, then you can try `Worker`, which allow you shut it down explicitly.